src/codegen.cpp

```diff
@@ -8209,42 +8209,20 @@ extern "C" void jl_init_llvm(void)
 #endif
 
     // Parse command line flags after initialization
-    const char *const argv_tailmerge[] = {"", "-enable-tail-merge=0"}; // NOO TOUCHIE; NO TOUCH! See #922
-    cl::ParseCommandLineOptions(sizeof(argv_tailmerge)/sizeof(argv_tailmerge[0]), argv_tailmerge, "disable-tail-merge\n");
-#if defined(_OS_WINDOWS_) && defined(_CPU_X86_64_)
-    const char *const argv_copyprop[] = {"", "-disable-copyprop"}; // llvm bug 21743
-    cl::ParseCommandLineOptions(sizeof(argv_copyprop)/sizeof(argv_copyprop[0]), argv_copyprop, "disable-copyprop\n");
-#endif
-#if defined(_CPU_X86_) || defined(_CPU_X86_64_)
-    const char *const argv_avoidsfb[] = {"", "-x86-disable-avoid-SFB"}; // llvm bug 41629, see https://gist.github.com/vtjnash/192cab72a6cfc00256ff118238163b55
-    cl::ParseCommandLineOptions(sizeof(argv_avoidsfb)/sizeof(argv_avoidsfb[0]), argv_avoidsfb, "disable-avoidsfb\n");
-#endif
-#if JL_LLVM_VERSION >= 120000
-    // https://reviews.llvm.org/rGc068e9c8c123e7f8c8f3feb57245a012ccd09ccf
-    Optional<std::string> envValue = sys::Process::GetEnv("JULIA_LLVM_ARGS");
-    if (envValue) {
-        SmallVector<const char *, 20> newArgv;
-        BumpPtrAllocator A;
-        StringSaver Saver(A);
-        newArgv.push_back(Saver.save("Julia").data());
-
-        // Parse the value of the environment variable into a "command line"
-        // and hand it off to ParseCommandLineOptions().
-        cl::TokenizeGNUCommandLine(*envValue, Saver, newArgv);
-        int newArgc = static_cast<int>(newArgv.size());
-        cl::ParseCommandLineOptions(newArgc, &newArgv[0]);
-    }
-#else
-    cl::ParseEnvironmentOptions("Julia", "JULIA_LLVM_ARGS");
-#endif
-
+    StringMap<cl::Option*> &llvmopts = cl::getRegisteredOptions();
+    const char *const argv[1] = {"julia"};
+    cl::ParseCommandLineOptions(1, argv, "", nullptr, "JULIA_LLVM_ARGS");
```

Comment:
Feeling kinda silly that I didn't see that being added when they removed the original option for parsing environment options...

Reply:
It wasn't added, so much as always there, so they had suggested we use it in their PR.